schemas/input/csl-citation.json

```diff
@@ -40,11 +40,9 @@
               "$ref": "#/definitions/locators"
             }
           },
-          "suppress-author": {
-            "type": ["string", "number", "boolean"]
-          },
-          "author-only": {
-            "type": ["string", "number", "boolean"]
+          "command": {
+            "type": "string",
+            "enum": ["author-in-text", "suppress-author", "author-only"]
```

Comment:
```suggestion
            "enum": ["regular", "author-in-text", "suppress-author", "author-only"]
```

Reply:
Why would we include "regular," aka "default," aka what the process should do in the absence of some other command?

If we did that, wouldn't we be requiring the command, else the process would fail with an error?